src/include/daos/drpc_modules.h

```diff
@@ -35,7 +35,8 @@
 
 enum drpc_module {
 	DRPC_MODULE_TEST		= 0,	/* Reserved for testing */
-	DRPC_MODULE_SECURITY_AGENT	= 1,
+	DRPC_MODULE_SRV			= 1,	/* daos_server */
```

Comment:
Doesn't matter right now since in development we're always deploying matching components, but in general it's better to add onto the end of the enum rather than changing the existing module order.

Reply:
OK, ..., I'll change this back to chronological ordering.